.github/workflows/deploy-preview.yml

```
@@ -0,0 +1,79 @@
+name: GitHub Actions Rain Solver Preview Deployment
+
+# on:
+#   pull_request:
+#     types: [opened, synchronize]
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)